src/main/java/service/parse/DataParserImpl.java

```diff
@@ -0,0 +1,27 @@
+package service.parse;
+
+import model.FruitRecord;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataParserImpl implements DataParser {
+
+    @Override
+    public List<FruitRecord> parseFruitRecords(List<String> lines) {
+        List<FruitRecord> transactions = new ArrayList<>();
+
+        for (int i = 1; i < lines.size(); i++) {
```

Comment:
This implementation assumes that the first line of the input is always a header and skips it. It would be safer to check the contents of the line to ensure it is a header before skipping it.